EntAppProject/src/main/resources/JBCreateAccount.html

```diff
@@ -1,53 +1,53 @@
-<!DOCTYPE html>
-
-<html>
-
-<head>
-
-<title>JobBook - Create an Account</title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
- 
-</head>
-
-<body>
- 
-<h1><span>Job</span>Book</h1>
-
-<h2>Create an Account</h2>
-
-<br>
-
-<fieldset>
-
-<form action="file:///C:/Users/Administrator/eclipse-workspace/jobBookFINAL/src/main/resources/templates/JBHomescreen.html" method="post">
-
-<div class="container">
-        <label for="useremail"><b>Email:</b></label>
-        <input type="email" placeholder="Email:" name="useremail" required>
-        <br>
-        <label for="username"><b>Username:</b></label>
-        <input type="text" placeholder="Username:" name="username" required>
-        <br>
-        <br>
-        <label for="userpassword"><b>Password:</b></label>
-        <input type="password" placeholder="Password" name="userpassword" required>
-        <br>
-        <label for="userpasswordconfirm"><b>Confirm Password:</b></label>
-        <input type="password" placeholder="Password" name="userpasswordconfirm" required>
-        <br>
-        <button type="submit">Create</button>
-        <label>
-        <br>
-        <input type="checkbox" checked="checked" name="remember"> Remember me?
-        </label>
-        </div>
-        <br>
-        <br>
-        </form>
- 
-</fieldset>
- 
-</body>
-
+<!DOCTYPE html>
```

Comment:
I am  not sure why you are using raw HTML and not the thymeleaf stuff, but even so, all of your 'source' should go under /src. Anything outside of /src is usually some kind of configuration file or something essential for builds.

Reply:
The same comment applies to the rest of the HTML

Reply:
Yes, definitely.  And remove the hardcoded path in the form submit... this will be meaningless once this is deployed somewhere!  Use relative path instead.